theano/tensor/nnet/corr.py

```diff
@@ -377,8 +374,8 @@ class CorrMM(BaseCorrMM):
         Set to `(1, 1)` to disable subsampling.
 
     """
-    def __init__(self, border_mode="valid", subsample=(1, 1), openmp=None):
-        super(CorrMM, self).__init__(border_mode, subsample, openmp=openmp)
+    def __init__(self, border_mode="valid", subsample=(1, 1)):
```

Comment:
Here, I was meaning to remove completly the \__init__ method. It need to keep the openmp parameter as before this commit. And the code end up being what the default \__init__ is when it is not implemented. So we don't need it.